go/cmd/dolt/commands/add.go

```diff
@@ -59,40 +61,74 @@ func (cmd AddCmd) ArgParser() *argparser.ArgParser {
 	return cli.CreateAddArgParser()
 }
 
+// Generate the query that will call the `DOLT_ADD` stored proceudre.
+// This is safe because the only inputs are flag names and validated table names.
```

Comment:
Comments on functions should conform to godocs.

// generateSql returns ... This function assumes validated/safe input, and does not error.

Reply:
Done